ow_static/plugins/base/js/attachments.js

```diff
@@ -26,6 +26,8 @@ var OWFileAttachment = function(params) {
                 '</div></div>');
 
         self.$previewCont.append(data['html']);
+        OW.trigger('base.attachment_rendered', {'data' : data});
```

Comment:
Unique identificator of the attachment instance should be passed to the event. E.g. this.uid